docs/latex/Inhalt/03_Inhalt/02_ddd.tex

```diff
@@ -0,0 +1,35 @@
+\chapter{Domain Driven Design}
+
+\section{Analyse der Ubiquitous Language}
+
+\paragraph{Sprache} Als Sprache der Ubiquitous Language standen Deutsch und Englisch zur Auswahl. Da die Domänenexperten die Domäne ebenso gut in englischer wie in deutscher Sprache beschreiben können, Englisch jedoch von wesentlich mehr Personen und Entwicklern verstanden wird als Deutsch, wurde Englisch als Sprache für die Ubiquitous Language gewählt.
```

Comment:
vielleicht als weiteren Grund, dass die Bedienung in englischer Sprache von den Endanwendern bevorzugt wird

Reply:
Ist korrigiert.